README.md

```diff
@@ -228,8 +228,8 @@ The following subcommands are supported:
                          Use -v or --verbose to display detailed `v8 inspect` output for each object.
                          Accepts the same options as `v8 inspect`
       findjsobjects   -- List all object types and instance counts grouped by typename and sorted by instance count.
-                         Requires `LLNODE_RANGESFILE` environment variable to be set to a file containing memory ranges for the
```

Comment:
Should we also update the source file that generated this?

Reply:
This line is wrapped in an #ifdef guard in the source, i.e., it doesn't print it if lldb >= 3.9.